test/models/driver_test.rb

```diff
@@ -61,11 +60,63 @@
   # Tests for methods you create should go here
   describe "custom methods" do
     describe "average rating" do
-      # Your code here
+
+      it "calculates accurate average rating for a driver" do 
+        # Arrange
+        new_driver.save
+        new_passenger = Passenger.create(name: "Kari", phone_num: "111-111-1211")
+        trip_1 = Trip.create(driver_id: new_driver.id, passenger_id: new_passenger.id, date: Date.today, rating: 5, cost: 1234)
+        trip_2 = Trip.create(driver_id: new_driver.id, passenger_id: new_passenger.id, date: Date.today, rating: 3, cost: 6334)
+        trip_3 = Trip.create(driver_id: new_driver.id, passenger_id: new_passenger.id, date: Date.today, rating: 2, cost: 6334)
+
+        # Act
+        average_rating = new_driver.average_rating(new_driver.id)
+
+        # Assert
+        expect(average_rating).must_equal 3.3
```

Comment:
This is a great and well-written test! I like that it's so clean and that the answer is right there (I'm glad that you just hard-coded the answer `3.3` instead of recalculcate it in the test).

It might be worth adding in a trip with a rating of `0`, or a test case that includes that.